src/themes/Default.jsx

```diff
@@ -270,39 +271,43 @@ export const config = {
     checkbox: {
       borderColor: colors.darkGray3,
       color: colors.darkGray2,
+      disabledOpacity: 0.6,
+
       sm: {
         height: '2.2rem',
         size: '1.4rem',
         top: '0.7rem',
         innerSize: '1.1rem',
-        paddingLeft: '2rem'
+        paddingLeft: '1rem'
       },
       md: {
         height: '2.7rem',
         size: '1.7rem',
         top: '0.4rem',
         innerSize: '1.4rem',
-        paddingLeft: '2.5rem'
+        paddingLeft: '1.5rem'
       },
       lg: {
         height: '3.2rem',
         size: '2.1rem',
         top: '0.1rem',
         innerSize: '1.8rem',
-        paddingLeft: '3rem'
+        paddingLeft: '2rem'
       }
     },
     radio: {
       borderColor: colors.darkGray3,
       backgroundColor: colors.darkGray2,
+      disabledOpacity: 0.6,
+
       sm: {
         height: '2.2rem',
         size: '1.4rem',
         top: '0.7rem',
         innerSize: '0.8rem',
         innerTop: '1rem',
         innerLeft: '0.3rem',
-        paddingLeft: '2rem'
+        paddingLeft: '1rem'
```

Comment:
These padding values don't seem right anymore, the checkbox is overlapping with the label.